docs/guides/dynamic-circuits-considerations.ipynb

```diff
@@ -56,7 +56,19 @@
     "- **Resets**.\n",
     "  - Central controller: Memory usage scales proportionally with the number of resets.\n",
     "- **Measurements**.\n",
-    "  - Central controller: Memory usage scales proportionally with the number of measurements used by the central controller for control flow."
+    "  - Central controller: Memory usage scales proportionally with the number of measurements used by the central controller for control flow.\n",
+    "\n",
+    "<Admonition type=\"note\" title=\"Buffer overflow error\">\n",
```

Comment:
Does this need to be a `note`? It's ultimately a limit on the measurement buffer, similar to other memory limits.

Reply:
Ah I see - so no need to bring it out with any more emphasis than the others on the page? I misunderstood and thought it was an addendum to the very last one specifically. I'll adjust it!

Reply:
done in 7244e3edbc14a31147938795a4878d2c88fe5e8e